.github/actions/yarn-install/action.yaml

```diff
@@ -36,3 +41,10 @@ runs:
         YARN_ENABLE_GLOBAL_CACHE: 'false'
         YARN_NM_MODE: 'hardlinks-local'
         HUSKY: '0'
+        YARN_ENABLE_HARDENED_MODE: >-
+          ${{
+            (github.repository_owner == 'Altinn' &&
+             ((github.event_name != 'pull_request' && github.event_name != 'pull_request_target' && github.event.repository.fork == false) ||
+              (github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false)))
+            && '0' || '1'
+          }}
```

Comment:
Docs say:
> the hardened mode is enabled by default when Yarn detects it runs in a pull request from a public GitHub repository

So this just serves to _disable_ hardened mode sometimes? I think it might be better to always have it on, unless there's some good reason not to? Why was this change needed?